tasks/release.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env sh
+
+release() {
+  # Clean node_modules
+  rm -rf node_modules && rm -f yarn.lock
+
+  # Install dependencies.
+  yarn
+
+  # Bump version.
+  yarn version --no-git-tag-version --new-version ${1:-patch}
+
+  # Get the new version number.
+  local version=`grep -m1 version package.json | awk -F: '{ print $2 }' | sed 's/[", ]//g'`
```

Comment:
This can be simplified:
```suggestion
  local version=`grep -oP '(?<="version": ")[\d.]+' package.json`
```